.github/workflows/package-windows.yml

```diff
@@ -154,12 +154,34 @@ jobs:
       - name: Build package
         shell: C:\cygwin64\bin\bash.exe --login '{0}'
         run: |
-          cd $GITHUB_WORKSPACE && eval $(opam env) && CC=x86_64-w64-mingw32-gcc.exe make package -kj$(nproc) && echo ""
+          cd $GITHUB_WORKSPACE && eval $(opam env) && CC=x86_64-w64-mingw32-gcc.exe make nuget-noversion -kj$(nproc) && echo ""
 
       - uses: actions/upload-artifact@v4
         with:
           path: |
-            everparse*.zip
-            everparse*.tar.gz
-            everparse*.nupkg
+            everparse
           name: everparse
+      - uses: actions/upload-artifact@v4
+        with:
+          path: |
+            EverParse*.nupkg
+          name: everparse-nupkg
+
+  test:
+    needs: build
+    runs-on: windows-latest
+    steps:
+      - name: Download and extract artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: everparse
+          path: ${{ github.workspace }}\test
+      - name: Check out repo
+        uses: actions/checkout@v4
+        with:
+          sparse-checkout: src\3d\tests
+          path: ${{ github.workspace }}\everparse
```

Comment:
I think you may not need all the `github.workspace` mentions here, it should be the directory where the runner is currently at. This is not so in the self-hosted runner only due to using a `--login` shell, I think.

Reply:
Thanks for your remark. However, I remember having issues with relative paths, so I inserted those absolute paths in a47be5ea0a7852709c6ef171ce8ca783ed226ba0.